sql/normandy_recipe_versions_v1.sql

```diff
@@ -0,0 +1,14 @@
+SELECT
+  f.key AS recipe_id,
```

Comment:
the use of `submission_date_s3 = @submission_date` in the where clause suggests that this query is meant to be incremental, so it needs to include `submission_date` in the output columns
```suggestion
  submission_date_s3 AS submission_date,
  f.key AS recipe_id,
```

Reply:
Thanks!